packages/plugins/src/cli/install/index.ts

```diff
@@ -9,28 +9,27 @@ import { sphinxUp } from './sphinxup'
 const installWithUpdate = async (
   spinner: ora.Ora,
   skipLibrary: boolean,
-  confirm: boolean
+  ci: boolean
 ) => {
-  if (!confirm) {
+  if (!ci) {
     await userConfirmation(
       'Would you like to install the Sphinx Foundry fork? This is required to use Sphinx and will replace your existing Foundry installation.\nConfirm? (y\\n):'
     )
-  }
+    const foundryupStatus = await spawnAsync('/bin/sh', [
```

Comment:
Yeah it would be better to use `/bin/bash`.

Reply:
Well, below is specifically a bash script vs. this is just a curl command, which is then piped into bash. The command below probably wouldn't work with `sh` (since it's just a link to whatever the system shell is), but this command probably would since it just depends on cURL being installed. That said, it's probably better to use `/bin/bash` for consistency.